packages/features/eventtypes/components/Locations.tsx

```diff
@@ -218,9 +218,10 @@ const Locations: React.FC<LocationsProps> = ({
 
           const option = getLocationFromType(field.type, locationOptions);
           return (
-            <li key={field.id}>
+            <li key={field.type}>
```

Comment:
field.id was a unique identifier given by useFieldArray. Everytime you press save button, it would give new field.id triggering a re render and it would have transition too as useAutoAnimate also depends on key. E2E was fast and would sometimes attempt to click button but due to transitions there would be 2 of the same type causing error.

![Screenshot 2024-09-01 180431](https://github.com/user-attachments/assets/a3b80b66-13d1-4fda-b14c-071688060b06)

Before:


https://github.com/user-attachments/assets/1916c731-219e-4af8-ad69-bd114608d471




After:


https://github.com/user-attachments/assets/b1c9b5b3-df2e-4365-a153-e78164f122c8